app/components/button/index.tsx

```diff
@@ -48,12 +58,22 @@ const Button = ({
             onPress={onPress}
             testID={testID}
         >
-            <Text
-                style={txtStyle}
-                numberOfLines={1}
-            >
-                {text}
-            </Text>
+            <View style={styles.container}>
+                {Boolean(iconName) &&
+                <CompassIcon
+                    name={iconName!}
+                    size={iconSize}
```

Comment:
Similar with size, not sure if we should just take the fontSize from the txtStyle, at least by default.

Reply:
the fontSize does not really match the icon size in the design